docling_core/types/doc/document.py

```diff
@@ -6,7 +6,7 @@
 """Models for the Docling Document data type."""
 
 from datetime import datetime
-from typing import Generic, Optional, Union
+from typing import Generic, List, Optional, Union
```

Comment:
Just FYI, since version 3.9 (minimum supported in `docling-core`) we can already use the python type `list` instead.